changelog.md

```diff
@@ -5,9 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [0.7.4]
 
 ### Changed
+
+- Added columns to the patches table
```

Comment:
I think this needs to be fixed - and you can put this addition under "unreleased" for now